### PR TITLE
Allows to change @sysurihash as the default contentIdKey in a Page View event.

### PR DESCRIPTION
**Pull request by francoislg**
## Quick description (1-2 lines)

Adds the ability to change the PageView event "contentIDKey" value
## Long description (details)

We needed to log stats using our own "contentIDKey" (not sysuri).
You can now define a default `DefaultPageViewField` in the main configuration, and you can send specific events with a specific page view directly in the PageView event arguments.

### CODE REVIEW

**Comment by francoislg** [comment]
@erocheleau Who is in charge of this bot..?

**Comment by erocheleau** [comment]
Me, sorry you can ping me on slack to explain what you need?



Étienne Rocheleau,
Demo Engineer

File: scenariolib/event_pageview.go [hunk 1]
```diff
@@ -46,6 +47,16 @@ func newViewEvent(e *JSONEvent) (*ViewEvent, error) {
 	}
 	event.clickRank = int(docNo)
 
+	if e.Arguments["pageViewField"] != nil {
+		if event.pageViewField, validcast = e.Arguments["pageViewField"].(string); !validcast {
+			return nil, errors.New("Parameter pageViewField must be of type string in ViewEvent")
+		}
+	} else if c.DefaultPageViewField != "" {
```
Comment (erocheleau) [inline scenariolib/event_pageview.go]: There is a file called defaults.go
You should add that check there instead so all the defaults values are kept in one place and easier to find.

Comment (erocheleau) [inline scenariolib/event_pageview.go]: Check also the method in `config.go` called `fillDefaults(c *Config)`
I follow the scheme that you just assign what values are present in the config received, and then fill in default values if some are needed but not provided.

**Pull request merged**